es-core/src/Log.h

```diff
@@ -5,8 +5,8 @@
 #include <sstream>
 
 #define LOG(level) \
```

Comment:
The LOG macro generates 18 warnings about dangling else...

/home/devel/temp/EmulationStation_Stable/es-core/src/HttpReq.cpp:127:4: warning: add explicit braces 
to avoid dangling else [-Wdangling-else]
                        LOG(LogError) << "Error removing curl_easy handle from curl_multi: " << curl_
multi_strerror(merr);
                        ^
/home/devel/temp/EmulationStation_Stable/es-core/src/Log.h:8:40: note: expanded from macro 'LOG'     
if(level > Log::getReportingLevel()) ; \
                                       ^
1 warning generated.